specification/datafactory/resource-manager/readme.az.md

```diff
@@ -0,0 +1,71 @@
+## AZ
+
+These settings apply only when `--az` is specified on the command line.
+
+``` yaml $(az)
+az:
+  extensions: datafactory
+  namespace: azure.mgmt.datafactory
+  package-name: azure-mgmt-datafactory
+az-output-folder:  $(azure-cli-extension-folder)/src/datafactory
+python-sdk-output-folder: "$(az-output-folder)/azext_datafactory/vendored_sdks/datafactory"
+
+input-file:
+- Microsoft.DataFactory/stable/2018-06-01/entityTypes/DataFlow.json
+- Microsoft.DataFactory/stable/2018-06-01/entityTypes/Dataset.json
+- Microsoft.DataFactory/stable/2018-06-01/entityTypes/IntegrationRuntime.json
+- Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json
+- Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json
+- Microsoft.DataFactory/stable/2018-06-01/entityTypes/Trigger.json
```

Comment:
why do we need these here?

Reply:
This is because a previous model missing issue in modelerfour  https://github.com/Azure/autorest.modelerfour/issues/220

Reply:
Basically it will not load poly-subclass's subclass if there's no explicit reference to that model

Reply:
We can also resolve this issue by add x-ms-discriminator-value in poly-subclass's subclass in the swagger to resolve this issue.